docs/install/redhat.rst

```diff
@@ -28,11 +28,27 @@ from the CrateDB package repository::
     # Import the public GPG key for verifying the package signatures.
     sudo rpm --import https://cdn.crate.io/downloads/yum/RPM-GPG-KEY-crate
 
-    # Register with the CrateDB package repository.
-    sudo rpm -Uvh https://cdn.crate.io/downloads/yum/7/x86_64/crate-release-7.0-1.x86_64.rpm
-
-The command above will install the ``/etc/yum.repos.d/crate.repo`` package
-repository configuration file.
+To register with the CrateDB package repository, create a file called ``cratedb.repo``
+in the ``/etc/yum.repos.d/`` directory for RedHat based distributions, or in the
+``/etc/zypp/repos.d/`` directory for OpenSuSE based distributions, containing::
+
+    [cratedb-stable]
+    name=CrateDB RPM package repository - $basearch - Stable
+    baseurl=https://cdn.crate.io/downloads/yum/7/$basearch
+    enabled=1
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-crate
+    autorefresh=1
+    type=rpm-md
```

Comment:
Those two parameters have not been part of the previous package repository configuration. They are now inherited from the advised package repository configuration [like ES is doing it](https://www.elastic.co/guide/en/elasticsearch/reference/current/rpm.html#rpm-repo).

We did not research about their meaning yet, please advise correspondingly if you think they should be removed again.

Reply:
On our tests, they didn't cause any harm, so we just followed ES here, and included them to the new configuration